README.md

```diff
@@ -1,2 +1,82 @@
 # queries
 Hosts reusable log queries which are built into a single queries.json file.
+
+
+# Query database structure
+
+```yaml
+queries:
+  query1:  # that is query id (sorted by id)
+    name: foo  # required
+    regex: bar  # optional
+    pattern: aaa # optional (either regex or pattern required)
+    url: ... # str or list[str], issue links to lp, bz,... (optional)
```

Comment:
either str or list of str, to avoid confusing and issues when parsing, it can't be both
why not to have both "lp", "bz" as fields like we have in skip list? also worth to have logstash url

Reply:
Because we would open pandora box: we would also need a jira, which jira? having a hardcoded instance does not scale, not even for low number of issue trackers.

TBH, I was not very keen on the duality string and list of strings but reality is:
* very common with ansible itself (if conditions?)
* over 9/10 of use cases are with a single string, i do not want to clutter the editing experience by forcing users to add braces in all cases because we have few exceptions
* when we compile the query definitions, we will convert these to computer friendly format, one that has a single data type. You do not need to worry about parsing *this* file from python or ansible, that is the source. We would load the built one which will be ready to loop over (without any complex logic).

We do not need logstash url because we do not know which logstash instance will perform these queries, still we can include a logstash url as a comment, but that would be only informative.

Also, we would be building the logstash query based on fields inside, so adding one would easily create a case where what we run is not the what is mentioned in the url. Anyway, we can reconsider the logstash-url aspect at the next iteration, not really needed of v1 MVP.